Source/missiles.cpp

```diff
@@ -1689,10 +1689,9 @@ void AddRing(Missile &missile, Point /*dst*/, Direction /*midir*/)
 
 void AddSearch(Missile &missile, Point /*dst*/, Direction /*midir*/)
 {
-	missile.var1 = missile._misource;

```

Comment:
Are you sure this `var1` is not read for this value at some other point after this call?

Reply:
checked now :P